prisma/schema.prisma

```diff
@@ -91,8 +102,8 @@ model ScheduledItem {
   approvedItem ApprovedItem @relation(fields: [approvedItemId], references: [id])
 
   // indexes
-  @@index ([scheduledSurfaceGuid])
   // Prevent scheduling the same item on the same Scheduled Surface for the same day multiple times.
   @@unique([externalId])
   @@unique([approvedItemId, scheduledSurfaceGuid, scheduledDate], name: "ItemScheduledSurfaceDate")
+  @@index([scheduledSurfaceGuid])
```

Comment:
nit pick: I think this go accidentally moved from line 105

Can be fixed later on!